cinema/views.py

```diff
@@ -1,6 +1,7 @@
 from datetime import datetime
 
 from django.db.models import F, Count
+from drf_spectacular.utils import extend_schema, OpenApiParameter
 from rest_framework import viewsets, mixins, status
 from rest_framework.authentication import TokenAuthentication
```

Comment:
do you need this import as you commented all auth classes? remove it and remove other imports you don't use